libmatekbd/matekbd-indicator.c

```diff
@@ -414,10 +410,7 @@ matekbd_indicator_prepare_drawing (MatekbdIndicator * gki, int group)
 			ln2cnt_map = NULL;
 		}
 
-		gtk_container_add (GTK_CONTAINER (align), label);
-		gtk_container_add (GTK_CONTAINER (ebox), align);
-
-		gtk_container_set_border_width (GTK_CONTAINER (align), 2);
```

Comment:
[GtkEventBox](https://developer.gnome.org/gtk3/stable/GtkEventBox.html) is a [GtkContainer](https://developer.gnome.org/gtk3/stable/GtkContainer.html), so you can simply use the same API as with the alignment: `gtk_container_set_border_width (GTK_CONTAINER (ebox), 2);`.  Or alternatively you can use CSS stuff, or the [GtkWidget](https://developer.gnome.org/gtk3/stable/GtkWidget.html) API to set [margin](https://developer.gnome.org/gtk3/stable/GtkWidget.html#GtkWidget--margin).